app/src/main/java/de/westnordost/streetcomplete/data/quest/VisibleQuestsSource.kt

```diff
@@ -72,13 +72,13 @@ import javax.inject.Singleton
         osmQuestSource.getAllInBBoxCount(bbox)
 
     /** Retrieve all visible quests in the given bounding box from local database */
-    fun getAllVisible(bbox: BoundingBox, questTypes: Collection<String>): List<QuestAndGroup> {
+    fun getAllVisible(bbox: BoundingBox, questTypes: Collection<String>): List<Quest> {
         if (questTypes.isEmpty()) return listOf()
         val osmQuests = osmQuestSource.getAllVisibleInBBox(bbox, questTypes)
         val osmNoteQuests = osmNoteQuestSource.getAllVisibleInBBox(bbox)
 
-        return osmQuests.filter(::isVisible).map { QuestAndGroup(it, QuestGroup.OSM) } +
-               osmNoteQuests.filter(::isVisible).map { QuestAndGroup(it, QuestGroup.OSM_NOTE) }
+        return osmQuests.filter(::isVisible).map { it } +
+               osmNoteQuests.filter(::isVisible).map { it }
```

Comment:
Isn't `.map { it }` a no-op?

Reply:
oops, you are right. This was some remains from the last implementation